src/js/dbus.js

```diff
@@ -387,6 +387,10 @@ function ServiceInterface(dbus, name, service) {
 ServiceInterface.prototype.makeCall = function(member, data) {
   var self = this;
   var metadata = self._methods[member];
+  if (!metadata || typeof metadata.handler !== 'function')
+    throw new TypeError(
+      `Invalid caller, ${member} are not defined at interface`);
```

Comment:
Since `metadata.handler` is check if is a function, maybe it's more reasonable to present a message that indicates `${member} is not callable`

Reply:
Fixed, thanks :)